elliott/elliottlib/cli/find_bugs_kernel_cli.py

```diff
@@ -253,26 +253,30 @@ def _new_jira_fields_from_bug(bug: Bug, ocp_target_version: str, kmaint_tracker:
         if kmaint_tracker:
             fields["labels"].append(f"art:kmaint:{kmaint_tracker}")
 
-        # TODO: The following lines are commented out because we haven't reached to agreement
-        # on how to handle kernel CVEs in OCP at this moment.
-        # Without the following lines, kernel CVEs will be copied as normal (non-CVE) bugs.
+        is_cve_tracker = set(constants.TRACKER_BUG_KEYWORDS).issubset(set(bug.keywords))
+        if is_cve_tracker:
+            # TODO: The following lines are commented out because we haven't reached to agreement
+            # on how to handle kernel CVEs in OCP at this moment.
+            # Without the following lines, kernel CVEs will be copied as normal (non-CVE) bugs.
+            # Find flaw bugs associated with the CVE tracker
+            #
+            # cve_flaws = []
+            # for flaw_id, flaw_bug in zip(bug.blocks, bug.bugzilla.getbugs(bug.blocks)):
+            #     if not flaw_bug:
+            #         raise IOError(f"Error getting flaw bug {flaw_id}. Permission issue?")
+            #     if not BugzillaBug(flaw_bug).is_flaw_bug():
+            #         continue  # this is not a flaw bug
+            #     cve_flaws.append(flaw_bug)
+            # labels = {"Security", "SecurityTracking"}
+            # cve_names = re.findall(r"(CVE-\d+-\d+)", bug.summary)
+            # labels |= set(cve_names)
+            # labels |= {f"pscomponent:{component}" for component in bug.components}
+            # labels |= {f"flaw:bz#{flaw.id}" for flaw in cve_flaws}
+            # fields["labels"] += sorted(labels)
 
-        # is_cve_tracker = set(constants.TRACKER_BUG_KEYWORDS).issubset(set(bug.keywords))
-        # if is_cve_tracker:
-        #     # Find flaw bugs associated with the CVE tracker
-        #     cve_flaws = []
-        #     for flaw_id, flaw_bug in zip(bug.blocks, bug.bugzilla.getbugs(bug.blocks)):
-        #         if not flaw_bug:
-        #             raise IOError(f"Error getting flaw bug {flaw_id}. Permission issue?")
-        #         if not BugzillaBug(flaw_bug).is_flaw_bug():
-        #             continue  # this is not a flaw bug
-        #         cve_flaws.append(flaw_bug)
-        #     labels = {"Security", "SecurityTracking"}
-        #     cve_names = re.findall(r"(CVE-\d+-\d+)", bug.summary)
-        #     labels |= set(cve_names)
-        #     labels |= {f"pscomponent:{component}" for component in bug.components}
-        #     labels |= {f"flaw:bz#{flaw.id}" for flaw in cve_flaws}
-        #     fields["labels"] += sorted(labels)
+            hint += "Please note that this bug is cloned as a non-CVE bug intentionally due to limitations in interoperability in internal processes.\n"
+
+        fields["description"] = f"{hint}\n----\n{bug.description}"
```

Comment:
hint already have `\n`

Reply:
It is fine. I intentionally added an extra \n to make a clear seperation.